saito-rust/src/saito/network_controller.rs

```diff
@@ -85,7 +85,7 @@ impl NetworkController {
         debug!("connecting to peer : {:?}", url);
         let result = connect_async(url.clone()).await;
         if result.is_err() {
-            warn!("failed connecting to peer : {:?}", peer);
+            warn!("failed connecting to peer : {:?}", peer); //TODO : Retry connecting after an interval
```

Comment:
you can setup clion to auto format with rustfmt. I think this line will break into 2 after formatting.